packages/console/src/assets/docs/guides/web-java-spring-boot/README.mdx

```diff
@@ -268,7 +258,7 @@ home.html:
 
 Create a new controller to handle the user page:
 
-```java
+```java title="UserController.java"
```

Comment:
could you move "Get user info" after "Checkpoint"?

Reply:
Rename the title to `Create user page`. Besides displaying user information, this section also handles the sign-out logic and the after sign-in redirect logic. Keeping it in the current order ensures a more fluent reading experience. .